content/_data/upgrades/2-204-1.adoc

```diff
@@ -59,13 +59,6 @@ java.lang.IllegalStateException: Expected 1 instance of jenkins.security.Resourc
 
 Remove the `downloadSettings` and `useBrowser` lines from the configuration as code file to resolve those errors.
```

Comment:
I wonder whether there are users of CasC who would be confused/overwhelmed by an instruction like "remove the `downloadSettings` entry". I think it's unlikely.

Reply:
I think you're right.  The phrasing would be simpler if it said:
```suggestion
Remove the `downloadSettings` entry from the configuration as code file to resolve those errors.
```

Is that what you were thinking or did I misunderstand?

Reply:
Right, I would basically assume working knowledge of YAML here and phrase it like this.